_data/canonical-tags/operation.yaml

```diff
@@ -0,0 +1,3 @@
+id: operation
+name: Operation
+description: Relevant to starting and maintaining the Kubernetes system itself.
```

Comment:
> the Kubernetes system

Can we replace this with "Kubernetes"? See `architecture.yaml` for the same question. So:

> description: Starting and maintaining Kubernetes.